.travis.yml

```diff
@@ -10,5 +11,8 @@ env:
 script:
   - make test config=coverage defines=$DEFINES -j2
   - make test config=release defines=$DEFINES -j2
+  - make test cxxstd=c++98 -j2
+  - make test cxxstd=c++03 -j2
+  - make test cxxstd=c++11 -j2
 
```

Comment:
The redundancy is intended. It is useful in case someone adds C++11isms or C++03isms but compatibility to a former C++ standard should be retained.